wordpress-mobile/WordPress-iOS
# Fix HUD message on saving/updating/scheduling/publishing a post

Fixes #6079 This PR updates the logic used to select the text displayed in the HUD during & after saving/updating/scheduling/publishing a post.

<img width="816" alt="napkin 37 02-15-17 1 32 36 pm" src="https://cloud.githubusercontent.com/assets/154014/22991831/405c2b2e-f383-11e6-9fc8-0d629a4d1d1c.png">

Note that the current post-post functionality should work as is.

**To test:**
1. Test in both the legacy and visual (non-aztec) editors

2. Create a new post and save.
2a. Update an existing post.
2b. Create a new draft.
2c. Update an existing draft.
2d. Schedule a post.

3. Verify the HUD displays the correct text.

4. Go into airplane mode. Test the items from step 2 again. Verify the proper error text is displayed in the HUD.

Needs review: @nheagy would you be so kind?
